FlightJournal.Web/Controllers/FlightController.cs

```diff
@@ -683,5 +666,16 @@ internal IEnumerable<int> GetLocationsAffectedByFlight(Flight flight)
             var affectedLocations = new[] { ClubController.CurrentClub.LocationId, flight.StartedFromId, flight.LandedOnId ?? 0, pilotClubLocationId, backSeatPilotClubLocationId, payerClubLocationId }.Distinct();
             return affectedLocations;
         }
+
+        public static bool UserCanEditFlight(FlightContext db, Guid flightId, IPrincipal user)
+        {
+            if (user.IsManager()) return true;
```

Comment:
IsEditor already incorporates the check for manager and administrator. 

Could you confirm if we have a retained that manager and editor is club specific, so the flight must be related to the club with pilot or location before they can edit it ?